### ISSUE TITLE
Frontend: ability to collapse long tables

### ISSUE THREAD

**Issue by dhimmel** [manubot/rootstock#215]
Would it be easy to add a button to collapse long tables?

_Originally posted by @slochower in https://github.com/manubot/rootstock/issues/209#issuecomment-487261171_ (relocated here)

**Comment by dhimmel** [comment]
@slochower any example manuscripts with a long or oversized table?

**Comment by slochower** [comment]
> @slochower any example (Manubot) manuscripts with a long or oversized table?

Pretty much any of them here, including Table 1.

https://slochower.github.io/smirnoff-host-guest-manuscript/v/ff89e7673fc8bab233594a294e51aad25d1c6839/

I'm pretty sure I've seen collapsible ones, but I can't recall where. Will keep my eyes out...

**Comment by vincerubinetti** [comment]
Sure we can do this fairly easily. We'll put it on the to-do list.

**Comment by dhimmel** [comment]
@vincerubinetti should we consider what additional types of interactivity users will often want? For example, what if a user wanted to use a javascript package like [datatables](https://datatables.net/) for additional interactivity, like sort and search? It's an open question to what extent we want to enable these features by default versus give users guidance on how to easily add them.

**Comment by vincerubinetti** [comment]
@dhimmel Perhaps we should open a general discussion about these types of third party plugins. So far we've been operating under the assumption of just Hypothesis. But there's probably a large set of plugins people might commonly use, and we should do our best to support them or even include them.

I guess I'm saying, let's create a new issue for discussing that more broadly. I'd need input from everyone on what plugins they think would be useful.

**Comment by slochower** [comment]
> @dhimmel Perhaps we should open a general discussion about these types of third party plugins.

Yup. Figuring out the best way to integrate, e.g., plot.ly would also be useful.
